ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -60,6 +60,50 @@ StackLayout {
             }
         }
  
+        ListModel {
+            id: suggestions
+        }
+
+        Connections {
+            target: chatsModel
+            onActiveChannelChanged: {
+              suggestions.clear()
+              for (let i = 0; i < chatsModel.suggestionList.rowCount(); i++) {
+                suggestions.append({
+                  alias: chatsModel.suggestionList.rowData(i, "alias"),
+                  ensName: chatsModel.suggestionList.rowData(i, "ensName"),
+                  address: chatsModel.suggestionList.rowData(i, "address"),
+                  identicon: chatsModel.suggestionList.rowData(i, "identicon"),
+                  ensVerified: chatsModel.suggestionList.rowData(i, "ensVerified")
+                });
+              }
+            }
```

Comment:
Alright, here's the deal:

We keep working with a `ListModel` as it's required by `SuggestionBox` but we mutate it with `suggestionList` data every time the active channel has changed (that's when contact data is re-fetched)

Reply:
Yeah, I did something similar for the accounts or something. It's annoying but it works